go/libkb/api.go

```diff
@@ -322,11 +322,15 @@ var lastUpgradeWarning *time.Time
 func (a *InternalAPIEngine) consumeHeaders(resp *http.Response) error {
 	u := resp.Header.Get("X-Keybase-Client-Upgrade-To")
 	p := resp.Header.Get("X-Keybase-Upgrade-URI")
+	m := resp.Header.Get("X-Keybase-Upgrade-Message")
```

Comment:
Hopefully URL-encoding would take care of binary stuff too. (We might care about that if we want to send terminal color, for example. Not that we *should*...) My worry is more that URL encoding/decoding might not be consistent from one platform to another. I know that JS has multiple functions for this, for example. One prefers `+` for spaces and one prefers `%20`? So any non-Go client parsing this header might end up having to reimplement Go's poorly documented escape function.